rfcs/0077-stale-issues-amendment.md

```diff
@@ -0,0 +1,135 @@
+---
+feature: stale-issues-amendment
+start-date: 2020-10-13
+author: blaggacao
+co-authors: (to be found)
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
+related-issues: https://github.com/NixOS/nixpkgs/pull/100460, https://github.com/NixOS/nixpkgs/pull/100462
+---
+
+# Summary
+[summary]: #summary
+
+RFC 51 implemented the stale bot with the following motivation:
+
+> We have a large number of open issues that have accumulated over the years. Not all of them are still valid and need our attention.
+> 
+> By marking stale issues, we can more easily filter issues for ones that have at least one person interested in them.
+
+Under this motivation, the inactivity period was chosen to be 180 days.
+
+This RFC proposes an alternative interpretation of "stale":
+
+_A stale issue (or PR) is an issue on which the discussion has went stale._
+
+_Explicitly, it has no other semantic than this. It is a neutral 
+aggregated indicator of facts. It reflects the fact that 
+individuals did choose not to interact on a particular issue or PR
+for an extended period of time._
+
+_180 days is too long of such extended period of time. People do relate
+"stale" with some shorter time frame. Therefore, a PR goes stale after
+a 90 days period (vs previously 180 days) and an issue goes stale after
+a 60 days period (vs previously 180 days)._
+
+# Motivation
+[motivation]: #motivation
+
+Under the renewed definition of _stale_, people are betrayed. 
+
+They are not told the truth, though the facts are long evident.
+
+A majority of issues or PRs with no interaction for quite short period
+of times already (maybe 3-6 weeks) has a lesser change of "success".
+
+People with vested interest should see the _stale_ label and be prompted
+to think: "Oh, I need this, too. Damn, it's stale. Let's have a look
+and do something to help out."
```

Comment:
I may be the only one, but if I stumble upon a problem (or want a new feature) I don't really care if an existing issue about it is stale or not, I'll try to provide some help to get it done.

Reply:
That's nice! But it does it invalidate the implicit argument about nor the benefits about transparency, does it?

Reply:
I wanted to express that I don't think that a `stale`-label has a benefit for this very case you described here - that one only decides to look at an issue they're affected by just because it's marked as stale now.

Reply:
You are absolutely right about this. But also imagine a scenario (not so remote):

- Filtering by stale label
- Skimming the titles
- Be prompted by the title something like: "Oh, this sounds interesting, but too bad, it's stale. Maybe it could need a hand?"

Or gaining a quick overview of the _staleness_ of the project, like to make a quick decision on weather to bet on the project or not.

Reply:
There are many more possible scenarios for illustrating the benefits of transparency. Maybe we could just short cut to agree that it's a good thing we should try to improve?

Reply:
I reworded in https://github.com/NixOS/rfcs/pull/77/commits/120af202277c13d6bb084adbb614464807496b8b

Hence marking this particular feedback as resolved. Thanks, the document is better now! Feel free to reopen, if there is anything more to it...

Reply:
If reformulated the motivation to better fir the purpose of this RFC. Thanks for prompting this possibility of misinterpretation. It is much clearer now. &rarr; https://github.com/NixOS/rfcs/pull/77/commits/609184fd539a1777a5250920b343b58b2a43c621

Feel free to un resolve, if there is anything more to it that I should take into consideration.